apps/rehabstod/src/components/Layout/Layout.tsx

```diff
@@ -15,7 +15,9 @@ export function Layout() {
     <div className="flex min-h-screen flex-col">
       <LayoutHeader />
       <main className="flex-1">
-        {config && config.banners.length > 0 && <GlobalAlert>{config.banners[0].message}</GlobalAlert>}
+        {config && config.banners.length > 0 && (
+          <GlobalAlert priority={config.banners[0].priority}>{config.banners[0].message}</GlobalAlert>
+        )}
```

Comment:
Should we only display the first available banner if there are more than one? Looking at the old client's code; it looks like it displays them all.

Reply:
I can add it but test said that only one banner could be added at a time in IA and in the requirements meeting we also talked about that we don't add more than one and don't want to show more than one.

Reply:
Alright, we can keep it like this, was just curious :)